disperser/batcher/txn_manager.go

```diff
@@ -183,6 +184,13 @@ func (t *txnManager) monitorTransaction(ctx context.Context, req *TxnRequest) (*
 			}
 			err = t.ethClient.SendTransaction(ctx, newTx)
 			if err != nil {
```

Comment:
The change lgtm. It looks to me this function is not just monitoring txn, but actually rolling it to the end (either failed, or succeeded w/ N confirmations).